pkgs/tools/package-management/nix/default.nix

```
@@ -234,13 +234,13 @@ in rec {
   nixUnstable = lib.lowPrio (callPackage common rec {
```

```suggestion
  nixUnstable = lib.lowPrio (callPackage common rec {
    # Update checklist
    #  - version
    #  - date
    #  - src rev/sha256
    #  - changes in upstream packaging
    #    git diff OLD..NEW -- $(find . -name \*.nix | grep -v test)
```

https://github.com/NixOS/nixpkgs/pull/136271/files#diff-b94b72b12e07bed8c4a3b4c50242499d5efb1e32797426d319574fd613bc07aeR235-R240